integration-tests/scenarios/grantDelegation.test.ts

```diff
@@ -1,29 +1,29 @@
 import "@frequency-chain/api-augment";
 import { ApiRx } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
+import { Codec } from "@polkadot/types/types";
 import assert from "assert";
-import { connect, createKeys } from "../scaffolding/apiConnection";
+import { connect } from "../scaffolding/apiConnection";
 import { createMsa, createProvider, createSchema, grantDelegation, grantSchemaPermissions, revokeDelegationByDelegator } from "../scaffolding/extrinsicHelpers";
-import { signPayloadSr25519 } from "../scaffolding/helpers";
+import { createAndFundAccount, DevAccounts, INITIAL_FUNDING, showTotalCost, signPayloadSr25519 } from "../scaffolding/helpers";
+
+describe("Delegation Scenario Tests", function () {
+    this.timeout(15000);
+
+    const context = this.title;
+    const source = DevAccounts.Alice;
+    const amount = INITIAL_FUNDING;
 
-describe("Delegation Scenario Tests", () => {
     let api: ApiRx;
     let keys: KeyringPair;
+    let schemaId: Codec;
 
     before(async () => {
         let connectApi = await connect(process.env.WS_PROVIDER_URL);
-        keys = createKeys("//Alice")
         api = connectApi
-    })
-
-    after(() => {
-        api.disconnect()
-    })
-
-    it("should grant a delegation to a provider", async () => {
```

Comment:
Does this need a `function` keyword?

Reply:
Does what? That's the diff from the old version, not the new code.